control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2207,6 +2207,10 @@ components:
           description: the drain state
           allOf:
             - $ref: '#/components/schemas/CordonDrainState'
+        node_nqn:
+          description: hosts allowed to connect the target
```

Comment:
Think you can remove the description and let it pick up the one from HostNqn automatically?

Reply:
Incorporated